Tests/Opc.Ua.Core.Tests/Types/Encoders/JsonEncoderBenchmarks.cs

```diff
@@ -331,4 +340,173 @@ public void GlobalCleanup()
         #endregion
     }
 
+    [TestFixture, Category("JsonEncoder")]
+    [SetCulture("en-us"), SetUICulture("en-us")]
+    [NonParallelizable]
+    [MemoryDiagnoser]
+    [DisassemblyDiagnoser(printSource: true)]
+    public class JsonEncoderEscapeStringBenchmark
+    {
+
+        [Params(1,2,3)]
+        public int StringVariantIndex  { get; set; } = 3;
+
+        [Benchmark]
+        public void EscapeStringBenchmark1()
+        {
+            m_memoryStream.SetLength(0);
+            m_memoryStream.Position = 0;
+            EscapedStringToStream(m_testString);
+        }
```

Comment:
I think without m_streamWriter.Flush()  the result might be misleading, because encoded bits are not written to the memory stream.

Reply:
moved flush from teardown to the Benchmark